docs/src/user-guide/command-line-interface.md

```diff
@@ -223,11 +208,14 @@ npx eslint --plugin eslint-plugin-mocha file.js
 
 #### `--rule`
 
-This option specifies the rules to be used. These rules are merged with any rules specified with configuration files. (You can use `--no-eslintrc` to change that behavior.) To define multiple rules, separate them using commas, or use the option multiple times. The [levn](https://github.com/gkz/levn#levn--) format is used for specifying the rules.
+This option specifies rules to be used.
 
-If the rule is defined within a plugin, you have to prefix the rule ID with the plugin name and a `/`.
+* **Argument Type**: Rules and their configuration specified with [levn](https://github.com/gkz/levn#levn--) format.
+* **Multiple Arguments**: Yes
 
-Examples:
+These rules are merged with any rules specified with configuration files. If the rule is defined in a plugin, you have to prefix the rule ID with the plugin name and a `/`.
```

Comment:
Perhaps we should keep the note about `--no-eslintrc` here?

Reply:
i think i removed the line b/c i didn't understand what it was saying, and figured it wasn't too important b/c it's in parenthesis. 

could you clarify what is meant by "You can use `--no-eslintrc` to change that behavior."? then i can add back in refactored, clearer form.

Reply:
Agree with @bpmutter, I think it’s safe to omit this.

Reply:
This was added in https://github.com/eslint/eslint/pull/2776 per https://github.com/eslint/eslint/issues/2773.

It explains how to run only the rules specified on the command line. Without `--no-eslintrc`, rules specified with config files will be run too. With `--no-eslintrc`, rules specified with `--rule` are not "merged with any rules specified with configuration files", meaning that the final list of rules to run is the list of rules specified with `--rule`.

I'm fine with removing the note, but it also makes sense to keep it here as it addresses one of the common use cases for `--rule`.

Reply:
I think @bpmutter's point, and I agree, is that the current text isn't clear as to what `--no-eslintrc` does in this context. It's way to vague to be able to understand your point. We can either remove the parens and leave it, or we can update the text to more explicitly describe the situation you mention, but I don't think leaving as-is makes sense.

Reply:
ok, thank you for the further clarification! 

i added a sentence to explain and example in 0365e98704475189f617e69c6bd000ae0b94e625